package.json

```
@@ -54,6 +60,7 @@
     "build:vanilla": "rollup -c --config-vanilla",
     "build:utils": "rollup -c --config-utils",
     "build:macro": "rollup -c --config-macro",
+    "build:macro-vite": "rollup -c --config-macro-vite",
```

and, change like this:
```suggestion
    "build:macro-vite": "rollup -c --config-macro_vite",
```